tests/e2e/configurer/upgrade.go

```diff
@@ -190,19 +190,20 @@ func (uc *UpgradeConfigurer) CreatePreUpgradeState() error {
 
 	wg.Add(6)
 
+	amountToFun := "10000000000000000000"
```

Comment:
just curious, why this name?

Reply:
Because I see that this func takes as input a tokensToFun ✌️ :

https://github.com/osmosis-labs/osmosis/blob/5fc1353fe118204df47e3884640949a24240ab2c/tests/e2e/configurer/chain/commands.go#L522-L524

Reply:
oh, I think it is should be named `amountToFund` then

Reply:
Thank you very much